src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -219,14 +221,28 @@ protected GraphQLObjectType resolveTypeForUnion(TypeResolutionParameters params)
         return result;
     }
 
-    protected ExecutionResult completeValueForEnum(GraphQLEnumType enumType, ExecutionStrategyParameters parameters, Object result) {
```

Comment:
Can we make the signature 
    protected ExecutionResult completeValueForEnum(GraphQLEnumType enumType, ExecutionContext 
 context, ExecutionStrategyParameters parameters, Object result) {

That way its sympathetic to the other methods that have ExecutionContext 
 context, ExecutionStrategyParameters parameters as parameters

In fact you could argue for consistency they should be the 1st two parameters (since this is a breaking change)